dashboard/README.md

```diff
@@ -1,70 +1,198 @@
+    │   │   │   │   └── NewRunsComponent.jsx
+    │   │   │   ├── PaginationComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── ProfileComponent
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── SidebarComponent
+    │   │   │   │   └── index.jsx
+    │   │   │   ├── TableComponent
+    │   │   │   │   ├── common-components.jsx
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   ├── TableOfContent
+    │   │   │   │   ├── common-components.jsx
+    │   │   │   │   ├── index.jsx
+    │   │   │   │   └── index.less
+    │   │   │   └── ToastNotificationComponent
+    │   │   │       └── index.jsx
+    │   │   └── containers
+    │   │       ├── Dashboard
+    │   │       │   └── index.jsx
+    │   │       └── MainLayout
+    │   │           ├── index.jsx
+    │   │           └── index.less
+    │   ├── reducers
+    │   │   ├── authReducer.js
+    │   │   ├── datasetListReducer.js
+    │   │   ├── endpointReducer.js
+    │   │   ├── index.js
+    │   │   ├── loadingReducer.js
+    │   │   ├── navbarReducer.js
+    │   │   ├── overviewReducer.js
+    │   │   ├── tableOfContentReducer.js
+    │   │   ├── toastReducer.js
+    │   │   └── userProfileReducer.js
+    │   ├── store
+    │   │   └── store.js
+    │   ├── utils
+    │   │   ├── axiosInstance.js
+    │   │   ├── dateFunctions.js
+    │   │   ├── filterDataset.js
+    │   │   ├── helper.js
+    │   │   └── routeConstants.js
+    │   ├── App.css
+    │   ├── App.js
+    │   ├── App.test.js
+    │   ├── index.css
+    │   ├── index.js
+    │   ├── logo.svg
+    │   ├── reportWebVitals.js
+    │   └── setupTests.js
+    ├── config-overrides.js
+    ├── jsconfig.json
+    ├── jsdoc.json
+    ├── package.json
+    ├── package-lock.json
+    └── README.md
+```
+
+## Assets
+
+Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
+
+## Prerequisites
+
+## Install Node JS
+Refer to https://nodejs.org/en/ to install nodejs
+
+## Install create-react-app
+Install create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
+
+```bash
+$ npm install -g create-react-app
+```
+
+## Cloning and Running the Application in local
+
+Clone the project into local
+Install all the npm packages. Go into the project folder and type the following command to install all npm packages
+
+```bash
+$ npm install
+```
+In order to run the application Type the following command
+
+```bash
+$ npm run dev
+```
+The Application Runs on localhost:3000
+
+## Build
+
+Build Application
+
+```bash
+$ npm run build
+```
+This will generate the `build` folder in the root directory, which contains packaged files such as `***.js`, `***.css`, and `index.html`.
+
+## Local Development
```

Comment:
This should probably be combined with the "Cloning and running the application" information above, although I like the "Local Development" header better.

Here you talk about the "passthrough" express server, but what's really important (covered above, indirectly) is that the express server is started up by the `npm run dev` script, and the `PBENCH_SERVER` environment variable is necessary to point at a real server.

Reply:
Moved the description to `cloning` section and updated the content of `Local development`